### ISSUE TITLE
getting a segmentation error(core dumped).. how to resolve this?

### ISSUE THREAD

**Issue by Abhinavkotha1** [RTimothyEdwards/irsim#6]
created a gate structure in magic and all extraction commands worked fine. when I try to invoke irsim, it shows some warnings and closes abruptly. Even the magic window gets closed and a segmentation error(core dumped) is displayed in terminal.

**Comment by RTimothyEdwards** [comment]
It sounds like you tried to run IRSIM from inside magic?  I haven't done that for a long time and I'm not sure whether it still works correctly or not---Part of the problem is that I'm not sure how IRSIM would pick up the correct setup file for the PDK that way.  Try generating a .sim file from magic and then running IRSIM as an independent program.  What versions of magic and IRSIM are you using?

**Comment by Abhinavkotha1** [comment]
Sir,
We are using magic 8.3 and irsim 9.7.
We used ext2sim command and irsim invoking command in same tkcon window
which is from magic.

**Comment by RTimothyEdwards** [comment]
@Abhinavkotha1 :  Yes, but what revisions?  Magic version 8.3 has had 402 revisions, while IRSIM 9.7 has had 116.
How are you calling IRSIM from magic?  What arguments are you passing to it?

**Comment by Abhinavkotha1** [comment]
Both are latest revisions. prm file and sim file are the arguments passing. Where can we get prm files for sky130A tech as they are not present in irsim folder.

**Comment by RTimothyEdwards** [comment]
But they are.  There is a whole set of PRM files for sky130A in open_pdks.  What version of the PDK are you using?
